static/styles/message.css

```diff
@@ -0,0 +1,120 @@
+$first_column_width: 46px;
```

Comment:
`46px` is we used right now in main too.

Reply:
Let's call this `avatar_column_width`; I think that more clearly expresses its purpose. (`first_column` might make you think of the app's left/middle/right columns).

Reply:
Ok done.